util/linux_install.sh

```diff
@@ -77,6 +77,10 @@ elif grep ID /etc/os-release | grep -q 'arch\|manjaro'; then
 		gcc \
 		git \
 		python \
+		python-pip \
+		python-appdirs \
+		python-argcomplete \
+		python-colorama \
```

Comment:
This is still not ideal IMO. Now, when we need to update the list of Python packages to install, it has to be done in two places. As I said, you should try to use the `--user` flag for pip to get around the permissions problems.